### PR TITLE
Add ShowConfigurations for all remaining session

### PR DESCRIPTION
**Pull request by aoiasd**
related:https://github.com/milvus-io/milvus/issues/18300
Signed-off-by: aoiasd [zhicheng.yue@zilliz.com](mailto:zhicheng.yue@zilliz.com)

### CODE REVIEW

**Comment by aoiasd** [comment]
rerun ut

File: internal/datacoord/server_test.go [hunk 1]
```diff
@@ -921,6 +921,38 @@ func TestServer_watchRootCoord(t *testing.T) {
...
+	atomic.StoreInt64(&svr.isServing, stateSave)
+
+	resp, err = svr.ShowConfigurations(svr.ctx, req)
+	assert.NoError(t, err)
+	assert.Equal(t, commonpb.ErrorCode_Success, resp.Status.ErrorCode)
+
+	for _, cfg := range resp.Configuations {
+		log.Info("ShowConfigurations",
+			zap.String(cfg.Key, cfg.Value))
+	}
```
Comment (jiaoew1991) [inline internal/datacoord/server_test.go:953]: should be assertion other than log

File: internal/datanode/data_node_test.go [hunk 2]
```diff
@@ -287,6 +287,36 @@ func TestDataNode(t *testing.T) {
...
+		node.State.Store(internalpb.StateCode_Healthy)
+
+		resp, err = node.ShowConfigurations(ctx, req)
+		assert.NoError(t, err)
+		assert.Equal(t, commonpb.ErrorCode_Success, resp.Status.ErrorCode)
+
+		for _, cfg := range resp.Configuations {
+			log.Info("ShowConfigurations",
+				zap.String(cfg.Key, cfg.Value))
+		}
```
Comment (jiaoew1991) [inline internal/datanode/data_node_test.go:317]: Ditto

**Comment by aoiasd** [comment]
@jiaoew1991 Modified

**Review by congqixia — APPROVED** [review]
/lgtm

**Pull request merged**